### PR TITLE
release MyLinkedList

### PR DESCRIPTION
**Pull request by u3IOm4uk**

### CODE REVIEW

File: src/main/java/core/basesyntax/MyLinkedList.java [hunk 1]
```diff
@@ -3,45 +3,161 @@
...
+        }
+        return -1;
+    }
+
+    private Node<T> getNode(int index) {
+        Node<T> current = first;
+        for (int i = 0; i < index; i++) {
+            current = current.next;
+        }
+        return current;
```
Comment (olekskov) [inline src/main/java/core/basesyntax/MyLinkedList.java:161]: Please recheck checklist
https://github.com/mate-academy/jv-linked-list/blob/master/checklist.md#dont-iterate-your-nodes-from-the-head-to-the-tail-or-vice-versa

File: src/main/java/core/basesyntax/MyLinkedList.java [hunk 2]
```diff
@@ -3,45 +3,161 @@
...
+            first = current.next;
+        } else if (current == last && size > 1) {
+            current.prev.next = null;
+            last = current.prev;
+        } else if (size > 1) {
+            current.next.prev = current.prev;
+            current.prev.next = current.next;
+        }
+        size--;
+        return removeValue;
```
Comment (olekskov) [inline src/main/java/core/basesyntax/MyLinkedList.java:96]: I cannot see any actions done if current size is 1 and passed index is 0, which is valid case

**Review by olekskov — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/MyLinkedList.java [hunk 3]
```diff
@@ -3,45 +3,175 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/MyLinkedList.java:6]:
```suggestion
```

File: src/main/java/core/basesyntax/MyLinkedList.java [hunk 4]
```diff
@@ -3,45 +3,175 @@
...
+    private static class Node<T> {
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/MyLinkedList.java:7]: move inner class to the bottom

**Review by kozhukhovsky — CHANGES_REQUESTED** [review]

**Pull request closed without merging**